test/spinxsk/spinxsk.c

```diff
@@ -1992,6 +2022,182 @@ ProcessPkts(
     return TRUE;
 }
 
+#define MAX_HEADER_STORAGE TCP_HEADER_STORAGE
```

Comment:
this definition might be slightly clearer, but directly using the underlying `TCP_HEADER_STORAGE` definition is preferrable IMO.